core/src/main/java/org/apache/calcite/rel/RelCollationTraitDef.java

```diff
@@ -16,11 +16,15 @@
  */
 package org.apache.calcite.rel;
 
+import org.apache.calcite.adapter.enumerable.EnumerableConvention;
+import org.apache.calcite.adapter.enumerable.EnumerableSort;
```

Comment:
this is not needed.

Reply:
Yes, will fix style issues and polish the code after people feel good about the general direction.